pkg/bosh/manifest/dns.go

```diff
@@ -165,14 +168,41 @@ func (dns *boshDomainNameService) Reconcile(ctx context.Context, namespace strin
 							VolumeMounts: []corev1.VolumeMount{
 								{MountPath: "/etc/coredns", Name: volumeName, ReadOnly: true},
 							},
+							ReadinessProbe: &corev1.Probe{
+								Handler: corev1.Handler{
+									HTTPGet: &corev1.HTTPGetAction{
+										Path:   "/health",
+										Port:   intstr.FromInt(8080),
+										Scheme: "HTTP",
+									},
+								},
+								FailureThreshold: 3,
+								PeriodSeconds:    10,
+								SuccessThreshold: 1,
+								TimeoutSeconds:   1,
+							},
+							LivenessProbe: &corev1.Probe{
+								Handler: corev1.Handler{
+									HTTPGet: &corev1.HTTPGetAction{
+										Path:   "/health",
+										Port:   intstr.FromInt(8080),
+										Scheme: "HTTP",
+									},
+								},
+								FailureThreshold:    5,
+								PeriodSeconds:       10,
+								SuccessThreshold:    1,
+								TimeoutSeconds:      5,
+								InitialDelaySeconds: 60,
+							},
```

Comment:
These are long enough (vertically in my editor) that it might be worthwhile splitting into separate methods… up to you though.

Does having a separate liveness probe make sense in this case?

Reply:
I copied the probes from the cluster CoreDNS instance. I prefer leaving it as is.